napari/layers/points/_tests/test_points.py

```diff
@@ -2418,3 +2419,21 @@ def test_empty_data_from_tuple():
     layer = Points(name="points")
     layer2 = Points.create(*layer.as_layer_data_tuple())
     assert layer2.data.size == 0
+
+
+@pytest.mark.parametrize(
+    'attribute, new_value',
+    [
+        ("size", [20, 20]),
+        ("face_color", np.asarray([0.0, 0.0, 1.0, 1.0])),
+        ("edge_color", np.asarray([0.0, 0.0, 1.0, 1.0])),
```

Comment:
Would be cool if we can add the same test for `edge_width` just for the sake of completeness!

Reply:
I agree!